spec/defines/sftp_jail_jail_spec.rb

```diff
@@ -7,7 +7,7 @@
         facts
       end
       let :pre_condition do
-        'include ssh'
+        'include ssh, sftp_jail'
```

Comment:
why is this change required? the defined resource already does an include for sftp_jail?

Reply:
In `sftp_jail::jail` I'm using variables from `sftp_jail` in default values. The `sftp_jail::*` variables were unknown without this change. Someone in the IRC pointed me in this direction.

Reply:
ah that's required because you assign them in the parameter section. so the class needs to be included before the defined resource gets evaluated. maybe this can be fixed by replacing the include in the defined resource with require. If that also doesn't work, the statement can be removed as it's redundant. The loading of sftp_jail needs to happen before the defined resources.

Reply:
I tried with require, but it didn't help. I'm happy to fix the problem a different way, but for now I did it again the same way (https://github.com/voxpupuli/puppet-sftp_jail/pull/74/commits/8d406e06ed2aa20b0f27bd5135578a7437f786be) as I don't exactly understand what you mean.